objects/src/lib.rs

```diff
@@ -43,6 +43,7 @@ pub mod assembly {
 }
 
 pub mod crypto {
+    pub use miden_crypto::hash;
```

Comment:
I think this is only needed in the context of the previous comment. So, if we switch there to importing `Digest`, this re-export will not be needed.